easybuild/easyblocks/generic/binary.py

```diff
@@ -98,6 +98,11 @@ def install_step(self):
         """Copy all files in build directory to the install directory"""
         if self.cfg['install_cmd'] is None:
             try:
+                cmd = self.cfg['preinstallopts']
+                if cmd:
+                    self.log.info("Running preinstallopts of %s using command '%s'..." % (self.name, cmd))
+                    run_cmd(cmd, log_all=True, simple=True)
+                self.log.info("Copying %s into '%s'..." % (self.cfg['start_dir'], self.installdir))
```

Comment:
Well, `preinstallopts` is for options that go *before* the install command.

For example `preinstallopts = 'PREFIX=%(installdir)s'`.